index.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface Style {
+	[key: string]: string;
+}
+
+export function getPropertyName(name: string): string;
+export function getStylesForProperty(name: string, value: string): Style;
+
+export default function transform(css: Array<[string, string]>, output?: string[], getStylesForProperty?: boolean): Style;
```

Comment:
I think this definition is wrong. The first argument is wrong, the second argument should be called `shorthandBlackList` and the third argument shouldn’t be there (I think)

Reply:
Thanks for reviewing.
1  what do you think about the first argument?
2 How can I understand 
> This can also be done by passing a third argument, `false` to `getStylesForProperty`.

 in readme?

Reply:
By the way, can I transform a whole file like this

```css
#id {
color: 'red';
}
.cls {
color: 'black';
}
```
@jacobp100

Reply:
@jacobp100 Are you still on this?

Reply:
```suggestion
export default function transform(css: Array<[string, string]>, shorthandBlacklist?: string[]): Style;
```

Reply:
Sorry for the delay. I've done what I think is correct. I don't use TypeScript, so could you double check? Thanks :D